rosdep/base.yaml

```diff
@@ -1586,26 +1586,56 @@ ignition-common3:
     buster: [libignition-common3-dev]
   ubuntu:
     focal: [libignition-common3-dev]
+ignition-common4:
```

Comment:
I know that at this point we're pretty much following the precedent from Citadel but what is the purpose of having a key name which does not match the debian package name? In this case, why is the rosdep key `ignition-common4` rather than `libignition-common4-dev`?

Reply:
Valid point, see the reasoning in this PR: https://github.com/ros/rosdistro/pull/25789#issue-448473195

Reply:
Right. Thanks for bringing that reference to the front!